static/js/around-map.js

```diff
@@ -0,0 +1,46 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoibmVvbWFkIiwiYSI6ImNqMHRrZ3ZwdzAwNDgzMm1kcHRhMDdsZGIifQ.bOSlLkmc1LBv0xAbcZXpog';
+const map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/streets-v9',
+  center: [-10, 45],
+  zoom: 2
+})
+
+// users interesting points
+const giveData = new Worker("/static/js/webworker-around.js");
```

Comment:
I would rename this to `pois`, or `workspots`, or something more related to what it actually is.